packages/contracts/tsconfig.json

```diff
@@ -15,6 +15,7 @@
     "./test",
     "./typechain",
     "./deploy",
-    "types/hardhat.d.ts"
+    "types/hardhat.d.ts",
+    "utils/metadata.ts"
```

Comment:
why not `./utils'` ?

Reply:
Fixed in 84a769252d3ffe2040c80a17a63a42ab79cb3a43